ElectronicObserver/Window/Dialog/EquipmentPicker/EquipmentDataPickerView.xaml

```diff
@@ -0,0 +1,186 @@
+﻿<common:WindowBase
+	x:Class="ElectronicObserver.Window.Dialog.EquipmentPicker.EquipmentDataPickerView"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:common="clr-namespace:ElectronicObserver.Common"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:dropRecordViewer="clr-namespace:ElectronicObserver.Window.Tools.DropRecordViewer"
+	xmlns:electronicObserverTypes="clr-namespace:ElectronicObserverTypes;assembly=ElectronicObserverTypes"
+	xmlns:local="clr-namespace:ElectronicObserver.Window.Dialog.EquipmentPicker"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:p="clr-namespace:ElectronicObserver.Properties.Window.Dialog"
+	xmlns:ui="http://schemas.modernwpf.com/2019"
+	x:Name="EquipmentPickerControl"
+	Title="{x:Static p:EquipmentPicker.Title}"
+	Width="800"
+	Height="450"
+	d:DataContext="{d:DesignInstance local:EquipmentPickerViewModel}"
+	ui:WindowHelper.UseModernWindowStyle="True"
+	x:TypeArguments="local:EquipmentDataPickerViewModel"
+	mc:Ignorable="d"
+	>
+	<Window.Resources>
+
+		<DataTemplate x:Key="AircraftProficiencyTemplate" DataType="{x:Type electronicObserverTypes:IEquipmentData}">
+			<common:AircraftProficiency AircraftLevel="{Binding AircraftLevel}" />
+		</DataTemplate>
+
+		<local:EquipmentPickerBindingProxy x:Key="Proxy" DataContext="{Binding}" />
+
+		<Style BasedOn="{StaticResource DefaultButtonStyle}" TargetType="Button">
+			<Setter Property="FontFamily" Value="{Binding DataContext.Font, Source={StaticResource Proxy}}" />
+			<Setter Property="FontSize" Value="{Binding DataContext.FontSize, Source={StaticResource Proxy}}" />
+			<Setter Property="Foreground" Value="{Binding DataContext.FontBrush, Source={StaticResource Proxy}}" />
+		</Style>
+		<Style BasedOn="{StaticResource DefaultCheckBoxStyle}" TargetType="CheckBox">
+			<Setter Property="FontFamily" Value="{Binding DataContext.Font, Source={StaticResource Proxy}}" />
+			<Setter Property="FontSize" Value="{Binding DataContext.FontSize, Source={StaticResource Proxy}}" />
+			<Setter Property="Foreground" Value="{Binding DataContext.FontBrush, Source={StaticResource Proxy}}" />
+			<Setter Property="Margin" Value="4 0" />
+			<Setter Property="Padding" Value="2 0" />
+			<Setter Property="MinWidth" Value="0" />
+		</Style>
+
+		<DataTemplate x:Key="EquipmentTemplate" DataType="{x:Type electronicObserverTypes:IEquipmentData}">
```

Comment:
Doesn't look like this gets used anywhere.

Reply:
Old code from first version